selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/taskmanagement/TaskManagementLocator.java

```diff
@@ -54,40 +80,52 @@
+          By.xpath(".//*[contains(@class,'addProject')]//*[contains(@name,'description')]");
 
   public static final By     ELEMETN_ADD_PROJECT_ENABLE_CALENDAR_CHECKBOX =
-                                                                          By.xpath(".//*[contains(@class,'addProject')]//*[contains(@class,'checkbox')]");
+          By.xpath(".//*[contains(@class,'addProject')]//*[contains(@class,'checkbox')]");
+  public static final SelenideElement ELEMENT_SAVE_PROJECT=$(byXpath("//*[@id=\"taskManagement\"]/div[4]/div/div[2]/form/div[7]/button[1]"));
 
   // Edit project form
-
+  public static final SelenideElement ELEMENT_POPUB_EDIT_PROJECT=$(byClassName("PopupContent"));
+  public static final By ELEMENT_ICON_PROJECT=byClassName("uiIconRightMenu");
+  public static final SelenideElement ELEMENT_EDIT_PROJECT_OPTION=$(byId("taskManagement")).find(byText("Edit"));
+  public static final SelenideElement ELEMENT_MENU_PROJECT=$(byClassName("uiDropdownWithIcon"));
+  public static final SelenideElement ELEMENT_BUTTON_SAVE=$(byXpath("//*[@id=\"taskManagement\"]/div[4]/div/div[2]/div[7]/button[1]"));
+  public static final SelenideElement ELEMETN_EDIT_PROJECT_ENABLE_CALENDAR_CHECKBOX=$(byClassName("uiCheckbox"));
+  public static final SelenideElement ELEMENT_EDIT_PROJECT =$(byXpath("//*[@id=\"taskManagement\"]/div[4]/div/div[2]/div[2]/span/div/form/div/div[1]/div/input"));
   // Share project form
 
   // Color
   public static final String ELEMENT_COLOR_TABLE_ITEM                     =
-                                                      ".//*[contains(@class,'project-name')][contains(.,'$project')]/..//*[contains(@class,'$color')]";
+          ".//*[contains(@class,'project-name')][contains(.,'$project')]/..//*[contains(@class,'$color')]";
 
   // Clone project
 
   // Show and Hide project
 
   // Delete project
-
+public static final SelenideElement ELEMENT_DELETE_PROJECT_OPTION=$(byId("taskManagement")).find(byText("Delete"));
+public static final SelenideElement ELEMENT_CONFIRM_DELETE=$(byClassName("confirmDeleteProject")).find(byText("Delete"));
   // *************************************Labels
   // Management*****************************************//
 
   // Labels-->General
+public static final SelenideElement ELEMENT_SAVE_EDIT_LABEL=$(byXpath("//*[@id=\"taskManagement\"]/div[4]/div/div[2]/div/div[2]/button[1]"));
+ public static final By ELEMENT_ICON_OPEN_MENU_LABEL=byClassName("uiIconRightMenu");
+ public static final SelenideElement ELEMENT_OPEN_MENU_EDIT_LABEL=$(byId("taskManagement")).waitUntil(Condition.appears, Configuration.timeout).find(byText("Edit"));
+    public static final SelenideElement ELEMENT_OPEN_MENU_DELETE_LABEL=$(byId("taskManagement")).waitUntil(Condition.appears, Configuration.timeout).find(byText("Delete"));
 
-  // Add labels form
+ // Add labels form
 
   // Edit labels form
```

Comment:
We just copied this class in this project from the old project, it was defined like this.
https://github.com/exoplatform/ui-test/blob/develop/ui-common/src/main/java/org/exoplatform/selenium/platform/TaskManagement/TaskManagementLocator.java